### ISSUE TITLE
Do not start wbar at login

### ISSUE THREAD

**Issue by novaTopFlex** [SpartanJ/wbar#91]
I have continuously noticed on my Linux system that wbar starts upon login to a given X session. I would like to stop this behavior, believing that this bar is annoying if it starts up at login. I have tested competing bars (polybar, yabar, etc.) and none start up upon login. Is there any way to stop this behavior where wbar is forced to start upon login?

**Comment by SpartanJ** [comment]
Hi! The repository doesn't provide any facility to auto-start wbar. This is usually controlled by your desktop environment, I don't know exactly how to help you but I can probably give you some hint if you provide me your desktop environment or display manager. Some desktop environments have an `autostart` folder where a `.desktop` file is located to enable the auto-start of an app. BTW this is just a fork of `wbar`, IDK if it's the same version that's distributed on your distro (can you tell me how did you install it and your distro?). Original wbar has been abandoned long ago, so I did fork it and added some new features.

**Comment by novaTopFlex** [comment]
Ubuntu 22.04 LTS: wbar is forced to autostart (my default desktop environment/DE is Openbox, but I remember the bar also autostarting with other Xsessions and supported Wayland sessions. How can I stop wbar from launching automatically at login to any Xsession? By the way, the command was “sudo apt install wbar” on Ubuntu which did not install the wbar-config package.

**Comment by novaTopFlex** [comment]
Sorry for the brief closure by the way. The above information (Ubuntu 22.04 LTS and apt-get) still holds true for my case.

**Comment by SpartanJ** [comment]
> Ubuntu 22.04 LTS: wbar is forced to autostart (my default desktop environment/DE is Openbox, but I remember the bar also autostarting with other Xsessions and supported Wayland sessions. How can I stop wbar from launching automatically at login to any Xsession? By the way, the command was “sudo apt install wbar” on Ubuntu which did not install the wbar-config package.

Checking the files provided in the [Ubuntu 22 wbar package](https://ubuntu.pkgs.org/22.04/ubuntu-universe-amd64/wbar_2.3.4-11_amd64.deb.html) there's not even a `.desktop` file, so the autostart does not come from the package but some configuration.

openbox does provider [several ways to autostart an application](http://openbox.org/wiki/Help:Autostart), but if you say that this is autostarting with other DEs then the most probable cause should be the DEs following the [XDG autostart spec](https://specifications.freedesktop.org/autostart-spec/autostart-spec-latest.html).

Basically try to locate any autostart over the most common autostart locations in XDG: `~/.config/autostart/`, `/etc/xdg/autostart/` and check if there's something at `~/.config/openbox/autostart`.

I won't be able to help you much more because this is not an issue with wbar, and I'm also not the author of wbar or the wbar packages provided in Ubuntu. But I hope you find a solution. Regards
